internal/v1/api.yaml

```
@@ -1483,7 +1483,7 @@ components:
       properties:
         ansible_controller_url:
           type: string
-          example: "example.towerhost.net"
+          example: "https://example.towerhost.net"
```

```suggestion
          example: "https://aap-gw.example.com"
```

This was a suggestion by someone on the #osbuild channel. Otherwise this seems like an improvement, too!

---

Cool, I'll update it quickly. I just took it from one of their scripts 😅